pisi/util.py

```diff
@@ -301,6 +301,21 @@ def join_path(a, *p):
             path += '/' + b
     return path
 
+def path_could_exists(path):
```

Comment:
This is a poorly named function..  And I really don't like the loop. Can't we just check for the permission issue itself when doing an lstat on the file?